lang/rustup/Makefile

```diff
@@ -0,0 +1,64 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rustup
```

Comment:
`rustup` is the name of the installer for Rust. I don't think this package should be named after the installer.

Reply:
On second thought, since this installs a `rustup` tool, perhaps the name is appropriate.